packages/create-sitecore-jss/src/initializers/react-native/prompts.ts

```diff
@@ -0,0 +1,14 @@
+import { DistinctQuestion, QuestionCollection } from 'inquirer';
+import { styleguidePrompts, StyleguideAnswer } from '../../common/prompts/styleguide';
+import { clientAppPrompts, ClientAppAnswer } from '../../common/prompts/base';
+
+export interface ReactNativeAnswer extends StyleguideAnswer, Partial<ClientAppAnswer> {
+  appName: string;
+  hostName: string;
+}
```

Comment:
Don't think we want Partial here (or duplicate props). Shouldn't this be:
```suggestion
export interface ReactNativeAnswer extends ClientAppAnswer, StyleguideAnswer {}
```